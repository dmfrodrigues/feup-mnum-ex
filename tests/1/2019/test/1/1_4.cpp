#include <bits/stdc++.h>
using namespace std;
double g1(double x){
  return (exp(x)*(x-1.0L)+11.0L)/(exp(x)-1.0L);
}
int main(){
  double x = 2.0L, x_;
  cout << setprecision(20) << fixed;
  for(int i = 0; i < 1000; ++i){
    x_ = x;
    x = g1(x);
    cout << x << endl;
    if(x == x_) break;
  }
  return 0;
}
