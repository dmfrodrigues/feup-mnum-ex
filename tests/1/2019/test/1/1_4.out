2.87821171299598788096
2.64557379933410752315
2.61150942900438964855
2.61086885967788040475
2.61086863814990222110
2.61086863814987601984
2.61086863814987601984
